packages/palette/src/elements/Select/Select.tsx

```diff
@@ -34,6 +35,11 @@ export const LargeSelect: SFC<SelectProps> = props => {
         disabled={props.disabled}
         onChange={event => props.onSelect && props.onSelect(event.target.value)}
       >
+        {placeholderActive(props) && (
+          <option selected value="" disabled>
```

Comment:
What happens if value is passed  in here _with_ `disabled`?

Reply:
It starts out selected, but after you choose something you can't go back to the placeholder value because it is disabled as an option.